i18n/cn/docusaurus-plugin-content-docs/current/faq/troubleshooting.md

```diff
@@ -46,4 +46,7 @@ logx.DisableStat()
 // client, _ := zrpc.NewClientWithTarget("127.0.0.1:8888")
 ```
 
+### 10. go-zero缓存默认是用db0? 可以redis配置一下用其它db吗?
+
```

Comment:
It seems an issue, you can post it under project go-zero, an FAQ must contains answer.

Reply:
See https://github.com/zeromicro/go-zero/pull/2214